examples/p4est_3d_dgsem/elixir_euler_source_terms_nonconforming_earth.jl

```diff
@@ -68,7 +68,7 @@ boundary_condition = BoundaryConditionDirichlet(initial_condition)
 boundary_conditions = Dict(:inside => boundary_condition,
                            :outside => boundary_condition)
 
-surface_flux = flux_hll
+surface_flux = FluxHLL(min_max_speed_naive)
```

Comment:
Similar to Eulergravity: Sufficiently special to justify the usage of `min_max_speed_naive`.